lib/new-ui/widgets/coins_page/action_row/coin_action_row.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:cake_wallet/entities/qr_scanner.dart';
+import 'package:cake_wallet/main.dart';
+import 'package:cake_wallet/new-ui/pages/send_page.dart';
+import 'package:cake_wallet/new-ui/pages/swap_page.dart';
+import 'package:cake_wallet/routes.dart';
+import 'package:cake_wallet/utils/feature_flag.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+import '../../../pages/receive_page.dart';
+import '../../../pages/scan_page.dart';
+import 'coin_action_button.dart';
+
+class CoinActionRow extends StatelessWidget {
+  const CoinActionRow({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal:18.0),
+      child: Row(
+        mainAxisSize: MainAxisSize.max,
+        mainAxisAlignment: MainAxisAlignment.center,
+        spacing: MediaQuery.of(context).size.width*0.05,
+        children: [
+          CoinActionButton(
+            icon: SvgPicture.asset("assets/new-ui/send.svg"),
+            label: "Send",
+            action: () {
+              if(FeatureFlag.hasNewUiExtraPages)
+              showModalBottomSheet(
+                context: context,
+                builder: (context) => SendPage(),
+              ); else Navigator.of(context).pushNamed(Routes.send);
```

Comment:
reformat the code please, we have a rule for the line length as 100, please set it in your IDE and reformat the code.
Also, please use curly brackets for if/else conditions `{}`